### PR TITLE
Exosuits' weapon fixes and exosuits' defines sorting

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
## Details
<details>
	<ul>
		<li> Exosuits' weapon now recharging from body's battery.
		<li> Exosuits now mortal.
	</ul>
</details>

### CODE REVIEW

**Comment by AlexMorgan3817** [comment]
Ready4Review, immortality of mechs' fix

File: code/modules/mechs/__mech_setup.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#include "_mech_defines.dm"
```
Comment (MadmanMartian) [inline code/modules/mechs/__mech_setup.dm:1]: No to all of this file.

Comment (AlexMorgan3817) [reply code/modules/mechs/__mech_setup.dm:1]: Unincluded file

File: code/modules/projectiles/guns/energy.dm [hunk 2]
```diff
@@ -84,18 +84,7 @@
...
-	if(istype(src.loc, /obj/item/rig_module))
-		var/obj/item/rig_module/module = src.loc
-		if(module.holder && module.holder.wearer)
-			var/mob/living/carbon/human/H = module.holder.wearer
-			if(istype(H) && H.back)
-				var/obj/item/weapon/rig/suit = H.back
-				if(istype(suit))
-					return suit.cell
-	return null
+	return loc.get_cell()
```
Comment (MadmanMartian) [inline code/modules/projectiles/guns/energy.dm:87]: I feel there are several ways this could be exploited

Comment (AlexMorgan3817) [reply code/modules/projectiles/guns/energy.dm:87]: Tell me some ways

Comment (AlexMorgan3817) [reply code/modules/projectiles/guns/energy.dm:87]: image

Comment (AlexMorgan3817) [reply code/modules/projectiles/guns/energy.dm:87]: It's mean that turfs can't return cell in get_cell() while it isn't coded

**Pull request merged**